settings.gradle

```diff
@@ -250,6 +250,9 @@ include ':instrumentation:struts-2.3:javaagent'
 include ':instrumentation:tomcat-7.0:javaagent'
 include ':instrumentation:twilio-6.6:javaagent'
 include ':instrumentation:undertow-1.4:javaagent'
+include ':instrumentation:vaadin-14.2:javaagent'
+include ':instrumentation:vaadin-14.2:vaadin-14.2-testing'
+include ':instrumentation:vaadin-14.2:vaadin-testing'
```

Comment:
is it possible to use `testSets` to be more consistent with how other instrumentations do multi-version testing?

Reply:
Done.